packages/appstore/src/public/images/index.ts

```diff
@@ -0,0 +1,4 @@
+import DemoLight from './wallet-demo-bg-light.svg';
```

Comment:
why we do not put these images in the `appstore/src/assets` ?

Reply:
@mahdiyeh-deriv It's because we can't use images for `background-image` from other path instead of src/public/images. It depends on `webpack` settings. I had the same problem, we tried to find a solution, but it works only when images for `backgound-image` are in this folder

Reply:
@mahdiyeh-deriv yeap, it related to webpack config, unfortunately i don't know how to setup webpack in order to use assets folder